src/components/Organization.vue

```diff
@@ -222,6 +226,25 @@ export default {
           console.log(e);
         });
     },
+
+    setGoogleMapString() {
```

Comment:
Lässt sich die Funktion so platzieren, dass wir sie nicht doppelt in der Codebase haben?